public/manifest.json

```diff
@@ -4,7 +4,6 @@
   "manifest_version": 2,
   "description": "Liquid Wallet Web Extension",
   "permissions": [
-    "alarms",
```

Comment:
why removing this permission?

Reply:
This PR does not use anymore the `alarms` API from the browser (periodic updater is now handled by a specific saga). So Marina doesn't need this permission

Reply:
how to periodic updater implements this? setInterval will be killed by the browser runtime.

Reply:
It uses the saga effect [delay](https://github.com/redux-saga/redux-saga/blob/24e9a68d621fd2a57a29b3b80e4b54ecb22fa593/packages/delay-p/src/index.js) (which do not use setInterval but setTimeout on each period).

Reply:
Reason why we used alarms in first place

https://developer.chrome.com/docs/extensions/mv3/migrating_to_service_workers/#alarms

Reply:
In fact, the periodic updater is only dispatching an action, I don't know if we need a specific service worker for this. Saga seems to perfectly handle timers in the background (it does not cancel the timers) and it offers a way to cancel easily via redux action

Reply:
rollback to browser.alarms done